web/src/invite.ts

```diff
@@ -474,6 +479,28 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
             set_streams_to_join_list_visibility();
         });
 
+        $("#invite_as").on("change", () => {
+            const invite_as_value = $("#invite_as").val();
+            const $stream_pill_container = $("#invite_streams_container .pill-container");
+            const $container: PillContainerJQuery = $stream_pill_container;
+            const pill_widget = $container.data("pill_widget");
+
+            if (!pill_widget) {
+                throw new Error("Pill widget not found");
+            }
+
+            if (invite_as_value === "600") {
```

Comment:
Instead of using the hardcoded value "600", it's best to use `settings_config.user_role_values.guest.code`.